pkg/database/mysql/env.go

```diff
@@ -10,6 +10,10 @@ func (m *MySQL) env() []corev1.EnvVar {
 			Name:  "SERVICE_NAME",
 			Value: m.ServiceName(),
 		},
+		{
+			Name:  "SERVICE_NAME_UNREADY",
```

Comment:
where do we use this service name?

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/12/files#diff-76e49ba3fd4f13d43ef2fc0f5461e24b60fccddbc9116e758bd3960dce6de0e6R47